utils/makefile.python

```diff
@@ -19,6 +19,7 @@ PY       ?=3
 PYTHON   ?= python$(PY)
 PIP      ?= pip$(PY)
 PIP_INST ?= --user
+PY_PRJ_REQ ?= requirements.txt
```

Comment:
when to use `make PY_PRJ_REQ=requirements-dev.txt pyenv` and not `make pyenvinstall` ?
(related to `-r requirements.txt` in `requirements-dev.txt` )

Reply:
The question should be: _In current master branch, where is `requirements-dev.txt` used?_ .. see https://github.com/searx/searx/pull/2524#discussion_r567641395

Reply:
I renamed this variable to `PYENV_REQ ?= requirements*.txt` .. a glob pattern matching requirement files.